CategoriesPanel.c

```diff
@@ -139,5 +159,6 @@ CategoriesPanel* CategoriesPanel_new(ScreenManager* scr, Settings* settings, Hea
    Panel_add(super, (Object*) ListItem_new("Display options", 0));
    Panel_add(super, (Object*) ListItem_new("Colors", 0));
    Panel_add(super, (Object*) ListItem_new("Columns", 0));
+   Panel_add(super, (Object*) ListItem_new("Header Format", 0));
```

Comment:
Header **Layout** ?
(and then s/Format/Layout/ig on the variable names / settings as well)